### PR TITLE
[TE] Self-Serve Onboarding 09: enhancements v1

### PR DESCRIPTION
**Pull request by newsummit**
Mainly taking care of the following:
- Making alert tuning severity default settings "sticky" (no blank field allowed)
- Time ranges are now synced between Alert Page and Tuning page
- Fixing a navigation issue (user was not able to get back to overview page - missing button)

### CODE REVIEW

File: thirdeye/thirdeye-frontend/app/pods/manage/alert/tune/route.js [hunk 1]
```diff
@@ -317,6 +318,14 @@ export default Route.extend({
       timeRangeOptions: setUpTimeRangeOptions([durationDefault], duration)
     });
     controller.initialize();
+
+    // Ensure date range picker gets populated correctly
+    later(this, function() {
+      controller.setProperties({
```
Comment (justYves) [inline thirdeye/thirdeye-frontend/app/pods/manage/alert/tune/route.js]: let's not mix old and new module apis. Good job on using `later` here, let's also use `get(this,...)` and `setProperties(controller, ....`

File: thirdeye/thirdeye-frontend/app/utils/manage-alert-utils.js [hunk 2]
```diff
@@ -282,6 +282,29 @@ export function buildAnomalyStats(alertEvalMetrics, anomalyStats, showProjected
   return anomalyStats;
 }
 
+/**
```
Comment (justYves) [inline thirdeye/thirdeye-frontend/app/utils/manage-alert-utils.js]: Per our conversation on slack, using local Storage is fine for launch. Eventually, we want to have more consistent code, and we'll move this into a service (a la rootcause)

File: thirdeye/thirdeye-frontend/app/utils/manage-alert-utils.js [hunk 3]
```diff
@@ -282,6 +282,29 @@ export function buildAnomalyStats(alertEvalMetrics, anomalyStats, showProjected
...
+ * Caches duration data to local storage in order to persist it across pages
+ * TODO: Move this to self-serve services
+ * @method setDuration
+ * @param {String} duration - qualifies duration set as 'custom' or a range key like '3m'
+ * @param {Number} startDate - start date for alert page (and list of anomalies)
+ * @param {Number} endDate - end date for alert page (and list of anomalies)
+ * @return {undefined}
+ */
+export function setDuration(duration, startDate, endDate) {
+  localStorage.setItem('duration', JSON.stringify({ duration, startDate, endDate }));
```
Comment (justYves) [inline thirdeye/thirdeye-frontend/app/utils/manage-alert-utils.js]: This should support all newer version of the browsers we use. Looks like there's no  need for a polyfill/addon at this point. 

screen shot 2018-02-14 at 10 24 49 am

File: thirdeye/thirdeye-frontend/app/utils/manage-alert-utils.js (hunk 2 above)
Comment (newsummit) [reply thirdeye/thirdeye-frontend/app/utils/manage-alert-utils.js]: Awesome. Looking forward to that. thanks.

File: thirdeye/thirdeye-frontend/app/utils/manage-alert-utils.js (hunk 3 above)
Comment (newsummit) [reply thirdeye/thirdeye-frontend/app/utils/manage-alert-utils.js]: Thanks for adding that note.

**Pull request merged**